app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkSearchControllerTest.kt

```diff
@@ -63,6 +74,8 @@ class BookmarkSearchControllerTest {
         val url = "https://www.google.com/"
         val flags = EngineSession.LoadUrlFlags.none()
 
+        assertNull(org.mozilla.fenix.GleanMetrics.BookmarksManagement.searchResultTapped.testGetValue())
```

Comment:
Can we `import org.mozilla.fenix.GleanMetrics.BookmarksManagement` so that it doesn't have to so verbose?